source/Classroom/Deliver/Undeliverable_Email/help_my_emails_arent_being_delivered.md

```diff
@@ -1,8 +1,8 @@
 ---
 seo:
   title: Help, My emails aren't being delivered!
-  description: Help, My emails aren't being delivered!
-  keywords: not, delivered, spam, reputation, getting, provisioning, provision, drop, inbox, email, out, report, client, bounce, invalid, deliverability, today, problem, delivery, clarification, complain
+  description: Learn about common reasons and solutions to why your message hasn't been delivered...
```

Comment:
Learn some common issues that lead to non-delivery and some solutions to help get your message in the inbox.